class_defines.py

```diff
@@ -2276,6 +2277,8 @@ def needs_wp(self):
 
     def use_flipping(self):
         # Only use flipping for constraint between point and line/workplane
+        if self.entity1.is_curve():
+            return False
```

Comment:
still need to get Flip working.  Hard to get Circle and Line to switch sides without this.